mongo/src/main/java/org/axonframework/extensions/mongo/eventhandling/saga/repository/MongoSagaStore.java

```diff
@@ -165,6 +178,7 @@ public static class Builder {
 
         private MongoTemplate mongoTemplate;
         private Supplier<Serializer> serializer;
+        private TransactionManager transactionManager = NoTransactionManager.instance();
```

Comment:
We typically describe the default of a value in the `static builder()` method and on the `Builder` class too.